agent/background.go

```diff
@@ -129,3 +131,22 @@ func (a *Agent) withCallbackTimeout(ctx context.Context, tc *taskContext) (conte
 	}
 	return context.WithTimeout(ctx, timeout)
 }
+
+func (a *Agent) startSpotTerminationWatcher(ctx context.Context, tc *taskContext, cancel context.CancelFunc) {
+	defer recovery.LogStackTraceAndContinue("spot termination watcher")
+	defer cancel()
```

Comment:
I can't imagine that we actually want to do this.

Reply:
True. Removed.